### PR TITLE
Refresher fixes

### PR DESCRIPTION
**Pull request by blairsteven**

### CODE REVIEW

File: test.c [hunk 1]
```diff
@@ -3475,24 +3475,75 @@ test_refresh_no_slash_callback (const char *path)
...
+    return _cb_timeout;
 }
 
 void
 test_refresh_no_slash ()
 {
-    const char *path = TEST_PATH"/interfaces/eth0/state";
+    const char *path = TEST_PATH"/interfaces/eth0";
 
     _cb_count = 0;
```
Comment (carlgsmith) [inline test.c:3486]: _cb_timeout = 5000;

File: test.c [hunk 2]
```diff
@@ -3475,24 +3475,75 @@ test_refresh_no_slash_callback (const char *path)
...
     CU_ASSERT (apteryx_refresh (path, test_refresh_no_slash_callback));
-    
+
     CU_ASSERT (apteryx_get (path) == NULL);
+    usleep(_cb_timeout * 2);
```
Comment (carlgsmith) [inline test.c:3490]: usleep (_cb_timeout); ?

File: test.c [hunk 3]
```diff
@@ -3475,24 +3475,75 @@ test_refresh_no_slash_callback (const char *path)
...
     CU_ASSERT (apteryx_get_tree (TEST_PATH"/interfaces/eth0") == NULL);
     CU_ASSERT (_cb_count == 2);
```
Comment (carlgsmith) [inline test.c:3492]: Actually, surely this test assertion has just changed. It should only be hit once and there should be not sleep above?

File: test.c (hunk 2 above)
Comment (blairsteven) [reply test.c]: Done

File: test.c (hunk 3 above)
Comment (blairsteven) [reply test.c]: Done

**Pull request merged**